app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
Ok, so I looked into it.  We're hardcoding VM in many places, most notably in the locale files. Note, below is the top count of hardcoded "VM" by file:

```
git grep -c -E "\s*[^#].+\bVM\b" | sort -rn -t ":" -k 2
config/locales/ja/manageiq.po:317
spec/tools/scvmm_data/get_inventory_output.yml:195
spec/tools/scvmm_data/get_inventory_output.xml:143
spec/tools/scvmm_data/get_inventory_output_hash.yml:141
config/locales/zh_CN/manageiq.po:139
config/locales/nl/manageiq.po:129
config/locales/manageiq.pot:129
config/locales/en/manageiq.po:129
spec/tools/vim_data/miq_vim_inventory/virtualMachinesByMor.yml:80
gems/pending/test/vms.yml:65
config/locales/en.yml:56
product/toolbars/x_vm_center_tb.yaml:55
product/toolbars/vm_center_tb.yaml:55
spec/tools/vim_data/miq_vim_perf_history/id2Counter.yml:54
config/toolbar_strings.rb:53
spec/vcr_cassettes/manageiq/providers/ansible_tower/configuration_manager/refresher.yml:51
db/fixtures/miq_event_definitions.csv:51
gems/pending/spec/db/MiqBdb/data/rpm/Packages:46
gems/pending/spec/db/MiqSqlite/conary.db:38
gems/pending/spec/db/MiqBdb/data/conary/conary.db:38
app/models/miq_action.rb:32
spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresher_3_1.yml:31
app/models/vm_or_template.rb:29
app/controllers/vm_common.rb:27
app/helpers/vm_helper/textual_summary.rb:26
spec/models/miq_expression/miq_expression_spec.rb:25
spec/controllers/vm_infra_controller_spec.rb:25
app/helpers/vm_cloud_helper/textual_summary.rb:25
app/helpers/vm_infra_helper/textual_summary.rb:24
db/fixtures/miq_product_features.yml:20
app/controllers/application_controller/compare.rb:20
spec/requests/api/vms_spec.rb:18
gems/pending/VMwareWebService/MiqVimVm.rb:18
db/fixtures/miq_alerts.yml:18
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Retirement/StateMachines/VMRetirement.class/__class__.yaml:16
db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/VMRetirement.class/__class__.yaml:16
app/models/miq_provision_virt_workflow.rb:16
gems/pending/spec/db/MiqBdb/data/rpm/Filemd5s:15
gems/pending/Verbs/implementations/SharedOps.rb:15
db/fixtures/ae_datastore/ManageIQ/Infrastructure/Cluster/Operations/Methods.class/__methods__/cluster_workload_management.rb:15
app/models/vm_or_template/operations.rb:15
gems/pending/spec/db/MiqBdb/data/rpm/__db.002:14
spec/models/miq_schedule_filter_spec.rb:12
product/charts/layouts/daily_perf_charts/VmOrTemplate.yaml:12
gems/pending/WriteVm/test/automateTest.rb:12
gems/pending/VMwareWebService/test/browserTest.rb:12
gems/pending/Scvmm/test/miq_scvmm_vm_ssa_info_test.rb:12
app/models/vm_scan.rb:12
spec/models/miq_alert_eval_internal_spec.rb:11
spec/helpers/application_helper/toolbar_builder_spec.rb:11
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Retirement/Email.class/__methods__/vm_retire_extend.rb:11
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Provisioning/StateMachines/VMProvision_VM.class/__class__.yaml:11
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Provisioning/Email.class/__methods__/miqprovision_complete.rb:11
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Operations/Methods.class/__methods__/vm_placement_optimization.rb:11
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Migrate/StateMachines/VMMigrate.class/__class__.yaml:11
db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/StateMachines/VMProvision_VM.class/__class__.yaml:11
db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/Email.class/__methods__/miqprovision_complete.rb:11
app/controllers/application_controller/ci_processing.rb:11
spec/models/vm_or_template_spec.rb:10
spec/models/miq_alert_spec.rb:10
gems/pending/VMwareWebService/test/eventHistoryTest.rb:10
gems/pending/VMwareWebService/test/cloneTest.rb:10
gems/pending/VMwareWebService/test/cloneDvsTest.rb:10
gems/pending/VMwareWebService/test/cloneCsmTest.rb:10
db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Retirement/Email.class/__methods__/vm_retirement_emails.rb:10
app/views/layouts/listnav/_storage.html.haml:10
app/models/vm_or_template/operations/relocation.rb:10
app/models/vm_or_template/operations/configuration.rb:10
app/helpers/storage_helper/textual_summary.rb:10
...
```

@martinpovolny @dclarizio @h-kataria if this is important to you, please open an issue, otherwise we can continue hardcoding it.  Thanks.